examples/next-notes/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@ceramicnetwork/http-client": "^2.3.4",
+    "@ceramicnetwork/http-client": "^2.3.5-rc.2",
```

Comment:
Don't bump to RCs or other non-stable versions please, that will create duplicate dependencies for consumers.